docs/wallet_API.md

```diff
@@ -71,6 +71,7 @@ const options = {
       },
     },
   },
+  enableMetrics: true,
```

Comment:
if we were to add `enableMetrics` option, think it'll worth explaining what that does to the suer reading this?